generator-jhipster-go/generators/server/templates/go/go/helper/HttpHelper.go

```diff
@@ -89,6 +89,6 @@ func MakePutCall(urlToPost string, body OperationBody, headers map[string]string
 		customlogger.Printfun("error","Error while making PUT call " + err.Error())    
 		return errors.New("Error while making PUT call " + err.Error()), nil
 	}
-	// customlogger.Printfun("info","Error while making PUT call " + "Successfull POST call with HTTP Status : " + resp.Status)    
+	customlogger.Printfun("info","Error while making PUT call " + "Successfull POST call with HTTP Status : " + resp.Status)    
```

Comment:
Successfull -> Successful
In the 2nd argument, `"Error while making PUT call " + "Successfull POST call with HTTP Status : "`, first concatenation is not required right, they can be put in one string -> `"Error while making PUT call Successful POST call with HTTP Status : "`

Reply:
sure!